KioskTests/CardHandlerTests.swift

```diff
@@ -40,9 +48,12 @@ class CardHandlerTests: QuickSpec {
             self.reader.fail = true
 
             var success = false
-            self.handler.cardSwipedSignal.subscribeError({ input -> Void in
-                success = true
-            })
+            self.handler
+                .cardStatus
+                .subscribeError { input -> Void in
```

Comment:
for things like this could you do 
``` swift
            self.handler
                .cardStatus
                .subscribeCompleted { success = true }
                .addDisposableTo(disposeBag)

```
?